### ISSUE TITLE
You just replace non-person pixels with their gray mode? 

### ISSUE THREAD

**Issue by Jason-xin** [huuuuusy/Mask-RCNN-Shiny#7]
You just replace non-person pixels with their gray mode?

**Comment by khanfarhan10** [comment]
Yes, in fact, we copy the original image to a template image where the masked numpy array is true. For many more effects you can checkout : https://github.com/khanfarhan10/advanced-color-splash ,
KUDOS!
